man/ukify.xml

```diff
@@ -427,6 +460,49 @@
         </varlistentry>
       </variablelist>
     </refsect2>
+    <title>Inspect configuration settings</title>
```

Comment:
Something is wrong here. `<title>` should appear immediately after a section is opened with `<refsect*>`.

Please preview the man page with `build/man/man ukify`.